en/frames_vtol/README.md

```diff
@@ -1,20 +1,127 @@
 # VTOL Airframes
 
-PX4 supports virtually all VTOL configurations (the complete set of supported configurations can be seen in [Airframes Reference > VTOL](../airframes/airframe_reference.md#vtol)):
-
-  * Tailsitters (duo and quadrotors in X and plus configuration)
-  * Tiltrotors (Firefly Y6)
-  * QuadPlane VTOL (standard plane plus quad)
-
-The VTOL codebase is the same codebase as for all other airframes and just adds additional control logic, in particular for transitions.
+PX4 uses the term VTOL to refer to vehicles that support both forward flight like a fixed-wing aircraft ("airplane") and vertical take off and landing like a helicopter or multicopter.
+
+![Quantum Vector: Tiltroto VTOL](../../assets/airframes/vtol/quantum_vector/hero.png)
+
+VTOL vehicles offer the benefits of both multicopter and fixed-wing flight:
+- **Vertical takeoff and landing:** Even inexperienced pilots can take off and land virtually anywhere.
+- **Fast and efficient fixed wing flight:** Faster, further, and longer missions, carrying heavier payloads.
+- **Hovering:** Steady platform for photography, structure scans etc.
 
 This section contains build logs and instructions for assembling and configuring a number of VTOL vehicle frames.
+
+## VTOL Types
+
+PX4 supports the three most important/main VTOL configurations.
+
+
+<style>
+.wrapper {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-auto-rows: minmax(100px, auto);
+}
+
+.grid_item {
+  border:1px solid gray;
+  border-radius: 5px;
+  background-color: #dfe2e5;
+  margin: 5px;
+  padding: 5px;
+}
+
+.grid_item_heading
+{
+  font-weight: bold;
+  margin-right: auto;
+  margin-left: auto;
+  padding: 5px;
+}
+
+.wrapper img {
+  margin-top: 10px;
+}
+</style>
+
+<div class="wrapper">

```

Comment:
Note this renders nicely in vuepress but not github. Sorry!

Reply:
It looks like this
![image](https://user-images.githubusercontent.com/5368500/126600739-ff14cbfe-6e2e-4e40-8eae-2b9eec902aff.png)

Reply:
>It looks like this

Love it!